lib/ruby2d/font.rb

```diff
@@ -0,0 +1,43 @@
+module Ruby2D
+  class Font
+    def self.all
+      all_full_paths
+        .map{|path| path.split("/").last.delete(".ttf").downcase }
```

Comment:
I'm not sure that this will work cross-platform (Windows). We could do `File.basename(path)` instead.